### PR TITLE
6.0.x fix#3790

### PR DESCRIPTION
**Pull request by 1letter**

### CODE REVIEW

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/controlpanel/tests/test_controlpanel_browser_usergroups.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+
+import transaction
+from plone.app.testing import SITE_OWNER_NAME, SITE_OWNER_PASSWORD
 from plone.testing.zope import Browser
 from Products.CMFCore.utils import getToolByName
 from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_FUNCTIONAL_TESTING
 from Products.CMFPlone.utils import normalizeString
 
-import transaction
-import unittest
```
Comment (mauritsvanrees) [inline Products/CMFPlone/controlpanel/tests/test_controlpanel_browser_usergroups.py]: The imports should have remained at this place.  Running either `isort` or `pre-commit run -a` should have fixed that.
But that is a detail, and it is not fully enforced yet in this repo I think.

Comment (mauritsvanrees) [reply Products/CMFPlone/controlpanel/tests/test_controlpanel_browser_usergroups.py]: Actually, let me run that on your branch.  Done and pushed.

**Review by mauritsvanrees — APPROVED** [review]
LGTM. Thanks!
One inline comment about isort, but I fixed that already on your branch.
Also, I changed the base target of your PR from master to 6.0.x.  I think a lot of people, including me, will make that mistake for a while. ;-)

@jenkins-plone-org please run jobs

**Pull request merged**